sp800-63b/sec5_authenticators.md

```diff
@@ -359,7 +359,7 @@ Biometric matching SHOULD be performed locally on claimant's device or MAY be pe
 
 If matching is performed centrally:
 
-* Use of the biometric SHALL be bound tightly to a single, specific device that is identified using approved cryptography.
+* Use of the biometric SHALL be limited to a single, specific device that is identified using approved cryptography.
```

Comment:
The whole point of server side is to enroll once, use many.  Otherwise, I would just enroll per device and use the local biometric.  I will talk to @enewt at 11 and give you a call after, but I think the requirement needs to change.